### ISSUE TITLE
New Player Guide Missing Sections

### ISSUE THREAD

**Issue by Turtlelord26** [trowl223/Compound_X#222]
Missing advisory sections on race, class, and background/skills. Pending scheduled, fundamental updates to all three areas.
- [ ] Added race advice
- [ ] Added class advice
- [ ] Added background and skills advice

**Comment by bleehu** [comment]
Can we break this into three separate tix? For instance, only backgrounds and skills advice is blocked by waiting for the backgrounds system. Only Race Advice is blocked by new race updates (though my hunch is that the new races won't actually change the advice much). And they don't really depend heavily on each other.

**Comment by Turtlelord26** [comment]
It's not new races, it's race updates that are blocking advice. Specifically racial traits and choosing them.
Will separate issue shortly.
